### PR TITLE
0.22 stable

### PR DESCRIPTION
**Pull request by Dynnammo**

### CODE REVIEW

**Comment by Dynnammo** [comment]
@armandfardeau I ran again tests on my forked branch and I got no errors. In local, I reproduced the error by deleting the env variable `POLIS_SITE_ID`, simply solved by exporting the `POLIS_SITE_ID` env variable.

I see no difference between the spec files from the 0.21-stable branch and here. **Can it be linked with the `POLIS_SITE_ID` secret that is not available on this pull request ?** Otherwise I really haven't a clue about why it's failing here...

File: spec/system/polis_spec.rb [hunk 1]
```diff
@@ -0,0 +1,65 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "pol.is", type: :system do
+  include_context "with a component"
+
+  let(:site_id) { ENV["POLIS_SITE_ID"] }
```
Comment (Quentinchampenois) [inline spec/system/polis_spec.rb:8]: Hi @Dynnammo, to me it is better to use static variables in tests to ensure a common behaviour

```suggestion
  let(:site_id) { "fake_site_id" }
```

Comment (Dynnammo) [reply spec/system/polis_spec.rb:8]: Agree. I'll check it by the end of the week.

**Comment by Quentinchampenois** [comment]
Hi, I close this PR because version is already present

**Pull request closed without merging**